project1/css/myIndex2.css

```diff
@@ -0,0 +1,116 @@
+* {
+    box-sizing: border-box;
+}
+
+/******************************************TOP***************************************/
+
+body {
+    margin: 25px 500px;
+}
+
+header {
+    /* border: solid yellow; */
```

Comment:
It's best practice to clean up comments when pushing code. 

This goes for random spacing, M.I.A. semi-colons, etc.